.github/workflows/pull-request.yml

```
@@ -462,3 +462,16 @@
+  Validate-Coderabbit:
+    name: Validate CodeRabbit Approval
+    runs-on: ubuntu-latest
+    if: github.actor != 'dependabot[bot]'
+    needs: [Code-Quality-Checks, Test-Application, Start-App-Without-Docker, Docker-Start-Check]
+    steps:
+      - name: Validate CodeRabbit.ai Approval
+        run: |
+          chmod +x .github/scripts/validate-coderabbit.sh
+          .github/scripts/validate-coderabbit.sh
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)